### DISCUSSION TITLE
Async-First Working Groups

### DISCUSSION THREAD

**Discussion by russelltrow** [Green-Software-Foundation/oc#69] [General]
@jawache, @russelltrow & @Sophietn would like to propose running the following experiment which would alter how the GSF's Working Groups operate. We very much welcome your comments.

**TL;DR**
We would like more asynchronous activity in the GSF, we would like to experiment with the Oversight Committee and then Working Groups moving to an Async-First approach.

# Summary of the Issue

Currently, synchronous meetings (meetings where everyone gets on a call simultaneously) are how the Foundation functions, our consensus model and voting require synchronous meetings. This is challenging for people who are not in the same time zone as the meeting for people who have a meeting clash.

Meetings are currently a core component of the functioning of GSF.

> A meeting-centric way of working on distributed teams can undermine deep work and flow, inclusion, flexible work and in the long run knowledge sharing. It also doesn’t lend itself to scale.
> [Adopting Asynchronous Collaboration in Distributed Software Teams](https://www.infoq.com/articles/asynchronous-collaboration-software-teams/)

![image](https://github.com/Green-Software-Foundation/oc/assets/11027021/2a7fac82-97bb-4f4a-984d-3076f00884b8)

### Meetings can be a blocker to collaboration
* Meeting are cancelled due to chairs being unavailable or low turnout on the day
* Meetings don’t always achieve quorum and are therefore unable to take decisions
* Meetings can happen at inconvenient hours for those outside Europe. Members who want to attend but are unable to, are left frustrated they are missing out on key discussions & updates
* Items might sit on an agenda without discussion for a period of time
* The pace of decision making is tied to the cadence of meetings
* Meetings can lack the preparation to be effective

### There is little engagement from the majority of the GSF membership
* Meetings have highly variable turnouts and consistently small numbers of attendees
* Meeting attendees are consistently the same people
* Those who can attend meetings take the majority of decisions

### What happens in meetings, stays in meetings
* Meetings are not documented comprehensively
* The outcomes of discussions are not always clear
* Actions agreed in meeting aren’t always tracked and followed up on

## Recent examples
* Cancelled: [2023.11.30 · Issue #88 · Green-Software-Foundation/standards-wg](https://github.com/Green-Software-Foundation/standards-wg/issues/88) 
```
Note: This was cancelled due to Thanksgiving
```
* Cancelled: [2023.09.19 · Issue #94 · Green-Software-Foundation/opensource-wg](https://github.com/Green-Software-Foundation/opensource-wg/issues/94)
* Cancelled: [[Cancelled] 2023-11-07 - Oversight Committee Agenda · Issue #60 · Green-Software-Foundation/oc](https://github.com/Green-Software-Foundation/oc/issues/60)
* No quorum: [2023-11-08 - Green Software Community Working Group Agenda · Issue #90 · Green-Software-Foundation/community-wg](https://github.com/Green-Software-Foundation/community-wg/issues/90) 
* Outcomes unclear: [2023-09-04 - Oversight Committee Agenda · Issue #47 · Green-Software-Foundation/oc](https://github.com/Green-Software-Foundation/oc/issues/47) 

## Our Hypothesis
Adopting a more asynchronous operating model for our Working Groups will have a positive impact on the the levels of engagement and contributions from across the GSF Member Organisations by removing the barriers to contributions imposed by physical meetings.

----

# Proposal
We would like more asynchronous activity in the GSF, we would like to experiment with the Oversight Committee and then Working Groups moving to an **Async-First approach**.

> Being "async-first" is not about being "async-only". It’s about recognising the benefits of asynchronous and synchronous communication patterns and being thoughtful about when you use either. The default, of course, is to start most collaborations, asynchronously. 
> [Adopting Asynchronous Collaboration in Distributed Software Teams](https://www.infoq.com/articles/asynchronous-collaboration-software-teams/) 

## Actions
- [ ] OC discuss this proposal in January and agree how to proceed
- [ ] Measure the current level of engagement and contribution
- [ ] Run a 2-3 month pilot with the Oversight Committee starting from February 2024
- [ ] Evaluate the pilot
- [ ] Identify the first Working Group to go Async

## This would mean
* Fostering a culture where meetings are a last resort, and ensuring that unavoidable meetings can be contributed to asynchronously
* Working Groups stop their regular meetings - extraordinary meetings are still possible
* All collaboration, discussions and decisions only take place on GitHub
* More regular asynchronous communication. Emails highlighting any opportunities for asynchronous collaboration. Announcement of all new motions and discussions
* An improved approach to remote decision-making and implementing the GSF governance model
* Being more [handbook-first](https://handbook.gitlab.com/handbook/handbook-usage/#why-handbook-first) so roles, responsibility, policies and procedures are clear for everyone
* Replacing verbal status updates with automated dashboards and reports
* More closely monitoring levels of engagement from WG Chairs and Contributors

### We won’t
* Change how the Steering Committee operates
* Change how Project teams operate, they can continue to choose how to run
* Make any changes without consulting with relevant stakeholders

### Measures of Success
* Increase in % of WG mailing list who contribute to GitHub Issues, Pull Requests or Discussions
* Increase in the diversity of contributors (Member Organisation, country, timezone)
* Increase in WG contributor satisfaction
* Decrease in decision-making time (TBC)
* Positive outcomes without a significant increase in Project Manager effort

---

# References

## Articles
* [Adopting Asynchronous Collaboration in Distributed Software Teams](https://www.infoq.com/articles/asynchronous-collaboration-software-teams/) 
* [The Future Of Work Is Asynchronous—And These Companies Are Leading The Way](https://blog.trello.com/the-future-of-work-is-asynchronous) 
* [10 companies that prioritize asynchronous communication.](https://www.modernteams.co/post/10-companies-that-prioritize-asynchronous-communication)
* [How to Move Your Team Toward Async-First Communication](https://async.twist.com/how-to-move-your-team-toward-async-first-communication/) 
* [The Art of Async: The Remote Guide to Team Communication](https://twist.com/remote-work-guides/remote-team-communication?ref=async.twist.com) 
  * [How async is your team? Take the Async Questionnaire to find out](https://async.twist.com/async-questionnaire/) 

## Guidance
* [A month with no meetings - An experiment to build an Async-First culture](https://assets.techsmith.com/docs/async-first.pdf)
* [The Async-First Playbook](https://www.thoughtworks.com/en-gb/insights/books/async-first-playbook)
* [Asynchronous agile | Go “async-first”](https://www.asyncagile.org/)
* [How We Work | 37signals Employee Handbook](https://basecamp.com/handbook/08-how-we-work#asynchronously)
* [The complete guide to asynchronous and non-linear working](https://handbook.gitlab.com/handbook/company/culture/all-remote/non-linear-workday/) 
  * [How to embrace asynchronous communication for remote work](https://handbook.gitlab.com/handbook/company/culture/all-remote/asynchronous/)
  * [The importance of a handbook-first approach to communication](https://handbook.gitlab.com/handbook/company/culture/all-remote/handbook-first/) 
  * [GitLab Handbook Usage](https://handbook.gitlab.com/handbook/handbook-usage/#why-handbook-first)

## Decision Making

* https://coda.io/@shishir/dory-pulse
* https://coda.io/@gokulrajaram/gokuls-spade-toolkit

**Comment by aecurrie** [comment]
I think this is a good idea. However, I also think we do get value from our
community group synchronous meetings. Maybe we should have regular purely
social synchronous catchups? It would be more efficient as we wouldn't have
to go through the processes and we could vary them early morning/late
afternoon?

**Reply by muemich** [reply]
I agree that this is a good idea and that there could be an element of a synchronous social meet. We would need to ensure that the social stays a social. But I do like the async idea a lot.

**Reply by jawache** [reply]
Totally, we're talking about regular social calls also, almost like what we used to have but even more social to ensure we still have a connection point with each other. Those calls should be at multiple times of the day to be inclusive of different timezones.

What would you want in those kinds of social calls?

**Comment by Sealjay** [comment]
I love the detail in here - I ask my team to read the GitLab Remote Work playbook, and take the certificate - [All Remote | The GitLab Handbook](https://handbook.gitlab.com/handbook/company/culture/all-remote/) - for all of the reasons you're listing, and I see it at the bottom of your guidance.
I'd suggest we have a calendar invite in (marked as free, with no Zoom, Google Meet or Teams call) as a forcing function/reminder to actually go in, review the proposals, and make some contributions.

Sounds like a great way forward.

**Reply by jawache** [reply]
That's such a good resource @Sealjay. @russelltrow I think we might all benefit from taking this course? Perhaps even add to our PM onboarding https://www.coursera.org/learn/remote-team-management

**Comment by aecurrie** [comment]
In the sync calls I really like the chance to discuss high level stuff:
what we are doing, what's working well, what we think needs improvement but
is that all stuff that should move to async?
